cypress/support/fragments/users/waiveFeeFineWarningModal.js

```diff
@@ -0,0 +1,23 @@
+import {
+  Button,
+  Modal,
+} from '../../../../interactors';
+
+const rootModal = Modal({ id: 'warning-modal' });
+
+const confirmButton = Button({ id: 'warningTransferContinue' });
+
+export default {
+  waitLoading:() => {
+    cy.expect(rootModal.exists());
+  },
+  uncheckDeselectToContinue:() => {
+    cy.get('#warning-mcl')
+      .find('[data-row-index="row-0"]')
+      .find('[class*="mclCell-"]:first-child').click()
```

Comment:
Please describe a reason why you don't use the interactor or add the TODO task to rewrite this piece of code using the interactors

Reply:
To click a specific cell - child of the first cell of each row